.github/workflows/update_tgs_dmapi.yml

```
@@ -0,0 +1,47 @@
+name: Update TGS DMAPI
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-dmapi:
+    runs-on: ubuntu-22.04
+    name: Update the TGS DMAPI
+    steps:
+    - name: Clone
+      uses: actions/checkout@v3
+
+    - name: Branch
+      run: |
+        git branch -f tgs-dmapi-update
+        git checkout tgs-dmapi-update
+        git reset --hard master
+
+    - name: Apply DMAPI update
+      uses: tgstation/tgs-dmapi-updater@v2
+      with:
+        header-path: 'code/__DEFINES/tgs.dm'
+        library-path: 'code/modules/tgs'
```

Incorrect `header-path`

```suggestion
    - name: Apply DMAPI update
      uses: tgstation/tgs-dmapi-updater@v2
      with:
        header-path: '__DEFINES/tgs.dm'
        library-path: 'code/modules/tgs'
```